### DISCUSSION TITLE
Is there a way to sort of compress the y-axis (start axis)?

### DISCUSSION THREAD

**Discussion by JoshLudahl** [patrykandpatrick/vico#515] [Questions] [answered]
I'm working with the latest alpha.5, is there a way to reduce the thickness and sort of compress the graph's start axis? I am reducing my y-axis to only use four values, so I want to compress the height of the chart so I when I add a few more charts, they don't take up as much screen space. It appears they have a default padding of some sort, is there a way to reduce that?

**Comment by patrickmichalik** [comment, marked as answer]
Hello! You can do this by reducing the height of the host (`CartesianChartHost` or `CartesianChartView`). The default is 200 dp, but you can resize the composable or the view as you normally would.

**Comment by JoshLudahl** [comment]
Thank you for your response, will check that out!

**Reply by patrickmichalik** [reply]
You’re welcome!
